java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -819,6 +819,40 @@ CompletableFuture<ClusterValue<Object>> fcallReadOnly(
      */
     CompletableFuture<ClusterValue<Map<String, Map<String, Object>>>> functionStats();
 
+    /**
+     * Returns information about the function that's currently running and information about the
+     * available execution engines.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/function-stats/">redis.io</a> for details.
+     * @return A <code>Map</code> with two keys:
+     *     <ul>
+     *       <li><code>running_script</code> with information about the running script.
+     *       <li><code>engines</code> with information about available engines and their stats.
+     *     </ul>
+     *     See example for more details.
+     * @example
+     *     <pre>{@code
+     * Map<String, Map<GlideString, Map<GlideString, Object>>> response = client.functionStatsBinary().get().getMultiValue();
```

Comment:
```suggestion
     * Map<GlideString, Map<GlideString, Map<GlideString, Object>>> response = client.functionStatsBinary().get().getMultiValue();
```

Reply:
Actually it should remain String. this is an primary ip and can never be bytes

Reply:
So the return value of the function should be changed as well